bin/raar-record-handler.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+################################################################################
+# raar-record-handler.sh - Moves new recordings to the raar import directory
+################################################################################
+#
+# Copyright (C) 2018 Radio Bern RaBe
+#                    Switzerland
+#                    http://www.rabe.ch
+#
+# This program is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public 
+# License as published  by the Free Software Foundation, version
+# 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License  along with this program.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+# Please submit enhancements, bugfixes or comments via:
+# https://github.com/radiorabe/raar
+#
+# Authors:
+#  Christian Affolter <c.affolter@purplehaze.ch>
```

Comment:
/me mumbles something about collective code owner ship

Not sure if this is relevant here though.

Reply:
Which is the case, the [code owner ship/copyright belongs to Radio Bern RaBe](https://github.com/radiorabe/raar/pull/1/files/4b43d9a3a1ee9489f0f0300627ef0eab135d85d3#diff-dc508681748c3bb80679a89a0674bb56R6)